images/linux/scripts/installers/snap-environment.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Why don't we use configure-environment.sh script?

Reply:
I though about it, but decided the semantic of "Configure environment" does not match exactly the task i am doing: "Complete installation of snap".

The putting the modification of snap to "configure environment" leads to 2 potential problems:

1. the installation of snap continues to be hidden from the provision template
2. there will be inconsistency with the other installed software: why most of them have its own scripts but snap does not

Both problems are insignificant and can be ignored, please ping me if i should use  configure-environment.sh script instead

Reply:
Snap is available on the system by default, we don't need to create additional script
https://snapcraft.io/docs/installing-snapd
![image](https://user-images.githubusercontent.com/48208649/79976703-bd61e080-84a5-11ea-8ba6-1c0aad7b2bea.png)